### DISCUSSION TITLE
rabbitmq mqtt plugin support $SYS topic ?

### DISCUSSION THREAD

**Discussion by qinfengge** [rabbitmq/rabbitmq-server#9975] [Questions] [answered]
I'm using the mqtt plugin and I'm wondering if the mqtt client is online and I see the $SYS topic mentioned in the mqtt specification. I want to know if the mqtt plugin in rabbitmq supports system topics
you can see info use this [link](https://github.com/mqtt/mqtt.org/wiki/SYS-Topics)

**Comment by michaelklishin** [comment, marked as answer]
RabbitMQ does not implement those metrics-related topics. See [Monitoring](https://www.rabbitmq.com/monitoring.html) and [use Prometheus](https://rabbitmq.com/prometheus.html), several identical or very similar metrics are provided by RabbitMQ regardless of the protocol used.
